docs/advanced-features/react-18/streaming.md

```diff
@@ -1,24 +1,15 @@
-# Streaming SSR (Alpha)
+# Streaming SSR
 
-React 18 will include architectural improvements to React server-side rendering (SSR) performance. This means you can use `Suspense` in your React components in streaming SSR mode and React will render them on the server and send them through HTTP streams.
-It's worth noting that another experimental feature, React Server Components, is based on streaming. You can read more about server components related streaming APIs in [`next/streaming`](/docs/api-reference/next/streaming.md). However, this guide focuses on basic React 18 streaming.
+React 18 includes architectural improvements to React server-side rendering (SSR) performance. This means you can use `Suspense` in your React components in streaming SSR mode and React will render them on the server and send them through HTTP streams.
+It's worth noting that the experimental React Server Components feature, is based on streaming. You can read more about server components related streaming APIs in [`next/streaming`](/docs/api-reference/next/streaming.md). However, this guide focuses on basic React 18 streaming.
 
-## Enable Streaming SSR
+## Use Streaming SSR
 
-Enabling streaming SSR means React renders your components into streams and the client continues receiving updates from these streams even after the initial SSR response is sent. In other words, when any suspended components resolve down the line, they are rendered on the server and streamed to the client. With this strategy, the app can start emitting HTML even before all the data is ready, improving your app's loading performance. As an added bonus, in streaming SSR mode, the client will also use selective hydration strategy to prioritize component hydration which based on user interaction.
+When you have Suspense in use in a SSR page, there is no extra configuration required to use streaming SSR.
 
-To enable streaming SSR, set the experimental option `runtime` to either `'nodejs'` or `'edge'`:
```

Comment:
The user doesn't need to use the Edge runtime? What happens if they use the Node.js runtime? Do they still get streaming?

Reply:
We should probably clarify this - you could stream using the Node.js runtime if you were self-hosting.

Reply:
No they don't need to "enable" streaming anymore, the rendering architecture is always enabled now. If they use the Edge runtime, they get streaming from the Edge. If they use the Node.js runtime, they will have streaming if Next.js is running in a Node.js server. And if it's running in a lambda function that doesn't support streaming, the response will be buffered and sent all at once when everything is ready.